src/providers/oauth-token-provider.js

```diff
@@ -0,0 +1,111 @@
+
+/**
+ * Module dependencies.
+ */
+
+import angular from 'angular';
+
+/**
+ * Token provider.
+ */
+
+function OAuthTokenProvider() {
+  var config = {
+    name: 'token',
+    options: {
+      secure: true
+    }
+  };
+
+  /**
+   * Configure.
+   *
+   * @param {object} params - An `object` of params to extend.
+   */
+
+  this.configure = function(params) {
+    if (params instanceof Object) {
+      angular.extend(config, params);
+    } else {
+      throw 'Invalid argument: `config` must be an `Object`.';
+    }
+
+    return config;
+  };
+
+  /**
+   * OAuthToken service.
+   *
+   * @ngInject
+   */
+
+  this.$get = function(ipCookie) {
+    class OAuthToken {
+
+      /**
+       * Get authorizationHeader.
+       */
+
+      get authorizationHeader() {
+        // return (this.tokenType && this.accessToken) ? `${this.tokenType} ${this.accessToken}` : undefined;
```

Comment:
[RFC 6749](https://tools.ietf.org/html/rfc6749#section-4.2.2) says that "Access Token Response" `token_type` is case insensitive. Not sure if this also means that the `token_type` inject in "Authorization Header" is also case insensitive.
Either way, can we rely on the `token_type` receive on the "Access Token Response" and sent it as is to the "Authorization Header"? Or should we transform the `token_type` to be sure that is a capitalised word?